services/src/datasets/external/nature40.rs

```diff
@@ -332,17 +351,56 @@ impl MetaDataProvider<GdalLoadingInfo, RasterResultDescriptor, RasterQueryRectan
             }
         })?;
 
-        Ok(Box::new(GdalMetaDataStatic {
-            time: None,
-            params: gdal_parameters_from_dataset(
-                &dataset,
-                band_index,
-                Path::new(&db_url),
-                None,
-                Some(self.auth().to_vec()),
-            )?,
-            result_descriptor: raster_descriptor_from_dataset(&dataset, band_index as isize, None)?,
-        }))
+        let (dataset, _band_labels, time_positions) = self
+            .get_band_labels_and_time_positions(dataset)
+            .await
+            .map_err(|e| geoengine_operators::error::Error::LoadingInfo {
+                source: Box::new(e),
+            })?;
+
+        let params = gdal_parameters_from_dataset(
+            &dataset,
+            band_index,
+            Path::new(&db_url),
+            None,
+            Some(self.auth().to_vec()),
+        )?;
+
+        let result_descriptor =
+            raster_descriptor_from_dataset(&dataset, band_index as isize, None)?;
+
+        if time_positions.is_empty() {
+            Ok(Box::new(GdalMetaDataStatic {
+                time: None,
+                params,
+                result_descriptor,
+            }))
+        } else {
+            let time_steps = time_positions
+                .iter()
+                .map(|time_position| {
+                    NaiveDateTime::parse_from_str(time_position, "%Y-%m-%dT%H:%M").unwrap()
+                })
+                .map(TimeInstance::from)
+                .map(|time_istance| TimeInterval::new_instant(time_istance).unwrap())
+                .collect::<Vec<_>>();
+
+            Ok(Box::new(GdalMetadataFixedTimes {
+                time_steps,
+                params,
+                result_descriptor,
+                time_placeholders: hashmap! {
+                    "%TIME%".to_string() => GdalSourceTimePlaceholder {
```

Comment:
you have to actually use the `%TIME%` placeholder somewhere. I guess in the `time` parameter of the WCS url in the `GetCoverage` request